generate-web/index.html

```diff
@@ -1,35 +1,74 @@
-<!doctype html>
-<title>Holo Config</title>
-<style>
-#form {
-  display: table;
-}
-
-#form li {
-  display: table-row;
-  list-style: none;
-}
-
-#form label {
-  padding-bottom: .5em;
-  padding-right: 1em;
-  display: table-cell;
-}
-
-#form input {
-  display: table-cell;
-}
-</style>
-<ul id="form">
-    <li>
-        <label for="email">Email</label>
-        <input type="text" id="email" name="email">
-    </li>
-    <li>
-        <label for="password">Password</label>
-        <input type="password" id="password" name="password">
-    </li>
-    <li>
-	<button id="generate">Generate</button>
-    </li>
-</ul>
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>Quickstart</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body class="step0">
+        <h1 class="stepH1 step0">Welcome to Holo</h1>
+        <h1 class="stepH1 step1 step2 step3">Let's start!</h1>
+        <h1 class="stepH1 step4">All done!</h1>
```

Comment:
TBH, I think this multi-modal flow is less user-friendly :(

Reply:
I'm leaving it to actual UX design - Celestial